compoments/report-repair/summary.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import SummaryList from '../summaryList';
+import Button from '../button';
+const Summary = ({values, changeLinkUrlValues}) => {
+  let [whatIsRepairProblemLink, setWhatIsRepairProblemLink] = useState(null)
+  let [describeRepairProblemLink, setDescribeRepairProblemLink] = useState(null)
+  useEffect(()=>{
+    setWhatIsRepairProblemLink(changeLinkUrlValues[values.repairLocation])
+    setDescribeRepairProblemLink(changeLinkUrlValues[values.repairProblem])
+    console.log('ADDRESS',values.address)
+  })
+  const personalDetailsSummary = [
+    {pageName:'Repair address', value: values.address+ ',' + values.postcode, link: 'postcode'},
+    {pageName:'Appointment contact number', value: values.contactPersonNumber, link: 'contact-person'},
+  ]
+  const repairDetailsSummary = [
+    { pageName:'Where is the problem?', value: values.repairLocation, link:'repair-location'},
```

Comment:
The state values for `repairLocation`, `repairProblem` and `repairProblemBestDescription` are the values that will be passed to the API, not the display values.
However, these are the values displayed on the summary page, which is incorrect.
The displayed value should be the same as the text for the option the user selected.